sandbox/package.json

```diff
@@ -17,6 +17,6 @@
     "lodash": "^4.17.20"
   },
   "devDependencies": {
-    "eslint": "^6.8.0"
+    "eslint": "^7.32.0"
```

Comment:
It looks fine to me. Just a question that Jack raised to upgrade eslint into 8.11.0 and couldn't do due to azure incompatibility. Do we have any ticket raised to investigate or upgrade azure version for future upgrade?

Reply:
I'll raise this with Dave Mays in our TRM catchup